packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -18,6 +20,14 @@ const propTypes = {
   * Array of links to show for the content of the help button
   */
   helpItems: PropTypes.array,
+  alerts: PropTypes.shape({
+    alertList: PropTypes.arrayOf(
+      PropTypes.shape({
+        messageID: PropTypes.string,
+        messageText: PropTypes.string }).isRequired,
```

Comment:
does making this required make the whole alert object required? I wouldn't think alerts would always be present